CONTRIBUTING.md

```diff
@@ -94,7 +94,11 @@ pytest tests/${TARGET_TEST_FILE_NAME}
 
 ## Continuous Integration and Local Verification
 
+Optuna repository uses GitHub Actions and CircleCI.
+
 CircleCI is used for continuous integration.
+Currently, we are migrating to GitHub Actions but still we use CirclCI for some tests listed below.
+`document` still exists because it makes it much easier to check built documentation and `tests-rdbstorage` is there because CircleCI better provides RDB storages.
```

Comment:
How about explaining in what way CircleCI is "better", if not for the record here on GitHub?

Reply:
"CircleCI currently supports more various RDB storages than GitHub Actions" sounds better to you?